templates/nutrition_score.tt

```diff
@@ -0,0 +1,41 @@
+
+<p>
+  <a data-dropdown="nutriscore_drop" aria-controls="nutriscore_drop" aria-expanded="false"> [% lang('nutriscore_calculation_details') %] &raquo</a>
+</p>
+<div id="nutriscore_drop" data-dropdown-content class="f-dropdown content large" aria-hidden="true" tabindex="-1">
+
+  [% IF is_beverage %]
+    <p> [% lang('nutriscore_is_beverage') %] </p>
+  [% ELSE %]
+    <p> [% lang('nutriscore_is_not_beverage') %] </p>
+  [% END %]
+  [% IF is_fat %]
+    <p> [% lang('nutriscore_proteins_is_added_fat') %] </p>
+  [% END %]
+
+  [% FOREACH group IN points_groups %]
+
+    <p> 
+      <strong> [% lang("nutriscore_" _ group.type _ "_points") %][% lang('sep') %]: [% group.points %]  </strong> 
+    </p>
+
+    <ul>
+      [% FOREACH nutrient IN group.nutrients %]
+        <li>
+          <strong> [% lang("nutriscore_points_for_" _ nutrient.id) %][% lang('sep') %]: [% nutrient.points %]&nbsp</strong>
```

Comment:
`[% lang("nutriscore_points_for_" _ nutrient.id) %]`

What's the `_` before `nutrient.id` for?

Reply:
@VaiTon it's for concatenation

Reply:
@VaiTon The underscore operator is explained at least on this part of the manual: http://template-toolkit.org/docs/manual/Directives.html#section_SET

Maybe the double quoted string interpolation would be clearer.

Reply:
> In Perl 5, the . is used for string concatenation, but in Perl 6, as in the Template Toolkit, the . will be used as the method calling operator and ' _ ' will be used for string concatenation.

I was used to the `.` for concatenation in Perl.

> Maybe the double quoted string interpolation would be clearer.
I agree. The others? @openfoodfacts/open-food-facts-contributors

Reply:
I just checked, we can do this instead:

[% lang("nutriscore_${group.type}_points") %]

[% lang("nutriscore_points_for_${nutrient.id}") %]

I think it's more clear than the [% lang("nutriscore_" _ group.type _ "_points") %] syntax.